samples/matter/common/src/bridge/bridge_storage_manager.h

```diff
@@ -23,40 +23,32 @@ namespace Nrf
  *		/brd_cnt/ /<uint8_t>/
  *		/brd_ids/ /<uint8_t[brd_cnt]>/
  * 		/brd/
- *			/eid/
- *				/0/	/<uint16_t>/
- *				/1/	/<uint16_t>/
- *				.
- *				.
- *				/n/ /<uint16_t>/
- *			/label/
- *				/0/	/<char[]>/
- *				/1/	/<char[]>/
- *				.
- *				.
- *				/n/ /<char[]>/
- *			/type/
- *				/0/	/<uint16_t>/
- *				/1/	/<uint16_t>/
- *				.
- *				.
- *				/n/ /<uint16_t>/
- *			/bt/
- *				/addr/
- *					/0/	/<bt_addr_le_t>/
- *					/1/	/<bt_addr_le_t>/
- *					.
- *					.
- *					/n/	/<bt_addr_le_t>/
+ *			/0/ /<BridgedDevice>/
+ *			/1/ /<BridgedDevice>/
+ *			.
+ *			.
+ *			/n/ /<BridgedDevice>/
+ *		/ver/ <uint8_t>
  */
 class BridgeStorageManager {
 public:
+	static constexpr auto kMaxUserDataSize = 128;
```

Comment:
```suggestion
	inline static constexpr auto kMaxUserDataSize = 128u;
```
Add `inline` if you're not defining that in .cpp file.